osu.Game.Modes.Taiko/Objects/Drawable/DrawableDrumRollTick.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Input;
+using osu.Game.Modes.Taiko.Judgements;
+using System;
+using osu.Game.Modes.Objects.Drawables;
+
+namespace osu.Game.Modes.Taiko.Objects.Drawable
+{
+    public class DrawableDrumRollTick : DrawableTaikoHitObject
+    {
+        private readonly DrumRollTick tick;
+
+        public DrawableDrumRollTick(DrumRollTick tick)
+            : base(tick)
+        {
+            this.tick = tick;
+        }
+
+        protected override TaikoJudgement CreateJudgement() => new TaikoDrumRollTickJudgement();
+
+        protected override void CheckJudgement(bool userTriggered)
+        {
+            if (!userTriggered)
+            {
+                if (Judgement.TimeOffset > tick.TickTimeDistance / 2)

```

Comment:
shouldn't this also be an abs check? just move this check to a bool outside both conditional areas and share it imo.

also may be better to add a tick.TickHitWindow and move the div2 there (or comment it better here too).